src/main/kotlin/io/hsar/wh40k/combatsimulator/logic/TacticalActionStrategy.kt

```diff
@@ -2,9 +2,24 @@ package io.hsar.wh40k.combatsimulator.logic
 
 import io.hsar.wh40k.combatsimulator.model.UnitInstance
 import io.hsar.wh40k.combatsimulator.model.World
+import io.hsar.wh40k.combatsimulator.model.unit.BaseStat
+import io.hsar.wh40k.combatsimulator.model.unit.StatUtils.getBonus
 
 object TacticalActionStrategy {
     fun decideTurnActions(world: World, thisUnit: UnitInstance): List<TurnAction> {
         TODO("Not yet implemented")
     }
+
+    fun canHalfMoveToEnemy(world: World, thisUnit: UnitInstance, enemy: UnitInstance): Boolean {
```

Comment:
I don't love the idea of putting these methods here - maybe it would look better in World where the position stuff already is? I don't know. It's not a huge problem as we can move stuff around, I'm just writing down my thought.

Reply:
I am very chilled about this, I think either works - I was trying to avoid stuffing too much stuff into the world-class by keeping anything that makes use of the world but isn't critical to world state tracking elsewhere. I think this falls into the category of 'uses world, but isn't core to world's functioning'. I can drop it into the world class for now if you like or create a utility class called MovementUtils or similar?

Reply:
I think that World at the moment has assumed responsibility for position tracking. If it gets really monolithic we can move to a MovementUtils class as you say, but for now I'd much prefer it to be in World.

Reply:
Overhauled into one movement checking method in world which uses a MoveAction interface as a param